ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-community-lists.j2

```diff
@@ -0,0 +1,26 @@
+{% if ip_community_lists is arista.avd.defined %}
+
+## IP Community-lists
+
+### IP Community-lists Summary
+
+| Name | Action | Communities / Regexp |
+| ---- | ------ | -------------------- |
+{%     for community_list in ip_community_lists %}
```

Comment:
Need to add a defined test for community_list.name as commented in eos template.

Similarly for **community_list.entries**, **entry.action**

Reply:
as I mentioned above - the idea was that it's mandatory